code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -229,12 +229,24 @@
 	cost = 3
 	no_coexistance_typecache = /obj/effect/proc_holder/spell/targeted/infinite_guns/arcane_barrage
 
+/datum/spellbook_entry/infinite_guns/Refund(mob/living/carbon/human/user, obj/item/spellbook/book)
```

Comment:
isn't there a variable to make a spell unable to be cast on the wizard ship?

I suppose this works too, though.

Reply:
Yeah but then new people can't test out the spell and see what it does
Kinda want to avoid that